src/projections/bipc.cpp

```diff
@@ -135,8 +138,10 @@ static PJ_LP bipc_s_inverse (PJ_XY xy, PJ *P) {           /* Spheroidal, inverse
         c = C45;
         Av = Azba;
     }
-    rl = rp = r = hypot(xy.x, xy.y);
-    fAz = fabs(Az = atan2(xy.x, xy.y));
+    r = hypot(xy.x, xy.y);
+    rl = rp = r;
+    Az = atan2(xy.x, xy.y);
+    fAz = fabs(Az);
```

Comment:
indeed, but where it is used in the loop, so depending on how smart the compiler, this might result in several invokations of fabs(Az) where Az isn't changed. I'll let as it is.